model/clusters_mgmt/v1/cluster_deployment_type.model

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of a clusterdeployment.
+class ClusterDeployment {
+	// Content of the clusterdeployment.
+	Content []Interface
```

Comment:
Won't this result in an array of `content` items with a single element?
```
{
  "kind": "ClusterDeployment",
  "href": "/api/clusters_mgmt/v1/clusters/1k1eiapjh11fpgtqig80ihj83rq93e0g/clusterdeployment",
  "id": "clusterdeployment",
  "content": [
    {
      "kind": "ClusterDeployment",
      "apiVersion": "hive.openshift.io/v1",
      ...
    }
  ]
}
```

Reply:
Whoops :facepalm:

Reply:
fixed. Nice catch